prediction_market_agent_tooling/deploy/agent.py

```
@@ -414,15 +410,13 @@ def before_process_market(
         """
         Executed before processing of each market.
         """
-        api_keys = APIKeys()
 
         if market_type.is_blockchain_market:
-            # Exchange wxdai back to xdai if the balance is getting low, so we can keep paying for fees.
+            # Ensure we have enough native token balance for transaction fees
             if self.min_balance_to_keep_in_native_currency is not None:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)